GUI/Arduino/UMSATS_GUI.ino

```diff
@@ -200,39 +251,48 @@ void sendGround(int command, int well,int seconds){
   outgoing.data.byte[6] = 0x00;
   outgoing.data.byte[7] = 0x00;
   
+  Can0.sendFrame(outgoing);
   
-if(command == TURN_ON_WELL || command == TURN_OFF_WELL){
-           Serial.print("Sent ground Message to ");
-          
-          if(command == TURN_ON_WELL) Serial.print("turn on well ");
-          else if(command == TURN_OFF_WELL) Serial.print("turn off well ");
-          else{Serial.print("ERROR !");}
-          
-          Serial.print(well);
-          Serial.print(" in ");
-          Serial.print(value);
-          Serial.print(" seconds.");
-          Serial.println();
-            }
-else if (command == DUMP_PAYLOAD_DATA){
-  
-  Serial.print("Sent ground message to dump payload data in ");
-  Serial.print(value);
-  Serial.println(" seconds.");
-  
+  if(command == TURN_ON_WELL || command == TURN_OFF_WELL){
+    
+    Serial.print("Sent ground Message to ");
+    
+    if(command == TURN_ON_WELL){
+      Serial.print("turn on well ");
+    }
+    else if(command == TURN_OFF_WELL){
+      Serial.print("turn off well ");
+    }
+    else{
+      Serial.print("ERROR !");
+    }
+    
+    Serial.print(well);
+    Serial.print(" in ");
+    Serial.print(seconds);
+    Serial.print(" seconds.");
+    Serial.println();
+    
+  }
+  else if (command == DUMP_PAYLOAD_DATA){
+    Serial.print("Sent ground message to dump payload data in ");
+    Serial.print(seconds);
+    Serial.println(" seconds.");
   }
   else if(command == GET_ADC_READING){
     Serial.print("Sent ground message to get reading from ADC ");
     Serial.print(well);
     Serial.print(" in ");
-    Serial.print(value);
+    Serial.print(seconds);
     Serial.print(" seconds");
     Serial.println();
-    }
-  Can0.sendFrame(outgoing);  }
+  }
   
-void printFrame(CAN_FRAME &frame) {
+}
   
+void printFrame(CAN_FRAME &frame) {
+  //This function preints out a CAN frame to the GUI.
```

Comment:
Typo: "prients" -> "prints". I'm not going to point all of them out, just mentioning that in general, you should make sure you use proper English (grammar and spelling), because it keeps the documentation professional and easier to read.